catalog/gke/nodepools/default/README.md

```diff
@@ -0,0 +1,30 @@
+# GKE Node Pool Package
+
+A kpt package to configure a GKE node pool.
```

Comment:
Is there a reason behind keeping this in `default`? Do we expect variants?

Reply:
This should probably be called `primary` not `default`.

Reply:
This allows you to add new node pools using the same blueprint, without requiring every user to explicitly clone the nodepool package.

Reply:
Why "primary"? It's default for this blueprint. It's just not default to GKE.

Reply:
Mostly because `default` is confusing with the GKE default node pool.

Generally, when multiple instances of something is possible, `primary` is our preferred term for the first/only instance.

Reply:
Can you point to some examples of that precedence? I've never seen that before.

Reply:
Can you explain why you're attached to `default` instead of `primary`?

Reply:
Anyways here's an example precedent: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/d60eec63598a8ff2d39505cd68268cba2af8843b/modules/beta-private-cluster-update-variant/cluster.tf#L22

Reply:
That's a whole cluster. It doesn't make sense to call that default because it's not automatically created when you create the project and it's in a cluster-specific module with no inherent parent module. I wouldn't call the Container Cluster "default". I used "example" instead.

In this case tho, this node pool IS the default node pool you get when installing the parent GKE package.

Reply:
You're right that the GKE module doesn't have additional cluster in it.

If I start with this module and later add another node pool (but leaving the original one unchanged), is that still the "default" node pool?

In this case, the strongest reason to not use the name "default" is that GKE already *has* a default node pool that you [are removing](https://github.com/GoogleCloudPlatform/blueprints/pull/16/files#diff-bf7eb17af94b0bd18fbe019600a9d75f4ad326988ae0aa5b467b288820ac34f3R7). Using the same term to refer to your custom initial node pool is confusing.

Reply:
renamed.